examples/seismic/tti/operators.py

```diff
@@ -354,7 +371,7 @@ def kernel_centered_3d(u, v, costheta, sintheta, cosphi, sinphi, space_order):
     """
     Gxx = Gxxyy_centered(u, costheta, sintheta, cosphi, sinphi, space_order)
     Gzz = Gzz_centered(v, costheta, sintheta, cosphi, sinphi, space_order)
-    return Gxx, Gzz
+    return second_order_stencil(model, u, v, Gxx, Gzz)
 
 
 def ForwardOperator(model, source, receiver, space_order=4,
```

Comment:
this is not a class method (correct? or am I being tricked by the diff?), so it should be spelled as `forward_operator`

Reply:
This is the main operator creation, so capitalized like all the others in acoustic/tti

Reply:
forgive me, I thought it wasn't part of a class